ReactTurboModuleCxx/React-TurboModuleCxx-RNW.podspec

```diff
@@ -33,6 +33,6 @@ Pod::Spec.new do |s|
   s.dependency "RCT-Folly", folly_version
   s.dependency "React-callinvoker", version
   s.dependency "ReactCommon/turbomodule/core", version
-  s.dependency "React-TurboModuleCxx-WinRTShared", version
-  s.dependency "React-TurboModuleCxx-WinRTPort", version
+  s.dependency "React-TurboModuleCxx-WinRTPort/Shared", version
+  s.dependency "React-TurboModuleCxx-WinRTPort/WinRT", version
```

Comment:
According to [the docs](https://guides.cocoapods.org/syntax/podspec.html#default_subspecs), you should be able to just specify the main spec and it should include all subspecs by default. I.e. can you try replacing the 2 lines with just this?

```ruby
  s.dependency "React-TurboModuleCxx-WinRTPort", version
```

Reply:
fixed